AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EViewHolder.java

```diff
@@ -99,9 +102,9 @@ private void createConstructorAndBuilder() {
 			}
 
 			JVar newCall = staticHelper.body().decl(narrowedGeneratedClass, "instance", newInvocation);
-			staticHelper.body().invoke(newCall, getOnFinishInflate());
+			OnFinishInflateCallBlock callBlock = new OnFinishInflateCallBlock(staticHelper.body().blockSimple(), body.blockSimple(), newCall);
```

Comment:
I think we should use `blockVirtual()` here to avoid unnecessary braces around the blocks.